scripts/test

```diff
@@ -51,7 +51,10 @@ compose() {
 (
   cd "${projectRoot}"
 
- # Guarantee clean environment
+  # create test output folder
+  mkdir -p test-output
```

Comment:
All our repos should really include this really to avoid needing to chmod `test-output` after first run.